### PR TITLE
Include analytics utility

### PR DESCRIPTION
**Pull request by ryanmparrish**
* Added analytics utility script to include tracking attributes to a given block
* Imported `decorateBlockAnalytics` into the media.js and `decorateLinkAnalytics` into the shared decorateText() utility 

Resolves: Hidden - no jira ticket

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/docs/library/blocks/media
- After: https://analytics--milo--adobecom.hlx.page/docs/library/blocks/media

**Testing notes**
 - Inspect the DOM to ensure the following data attributes are included. (daa-im, daa-lh, daa-ll) see utils/analytics.md for more details. 
 - FYI these are opt in. Needed blocks will require an include update to add these in their constructor.

### CODE REVIEW

**Comment by ryanmparrish** [comment]
@auniverseaway  - This is ready for review. Should we include this on all our existing blocks by default? 
(Marquee, Text, icon, zpattern, media).. may be missing a few.

File: libs/utils/analytics.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export function decorateBlockAnalytics(blockEl) {
+  const lh = [];
+  const exclude = ['--', 'block'];
+  blockEl.classList.forEach((c) => {
+    if (!c.includes(exclude[0]) && c !== exclude[1]) lh.push(c);
+  });
```
Comment (chrischrischris) [inline libs/utils/analytics.js:6]: nit: inline exclude

```suggestion
  blockEl.classList.forEach((c) => {
    if (!c.includes('--') && c !== 'block') lh.push(c);
  });
```

**Comment by ryanmparrish** [comment]
Apologies, looks like this branch was out of date. Guess that's what reviews are for.
I have updated this w/ some minor refactor. Give it a gander, sorry for the hickup.

File: libs/utils/analytics.js (hunk 1 above)
Comment (ryanmparrish) [reply libs/utils/analytics.js]: @chrischrischris  - Sorry, see my comment below. Good catch. This branch was out of date. I have updated this now if you could have another look

**Review by chrischrischris — APPROVED** [review]

File: libs/utils/analytics.js (hunk 1 above)
Comment (auniverseaway) [reply libs/utils/analytics.js]: @ryanmparrish we can remove the double dash check. With Helix 7, double dashes are removed.

Comment (ryanmparrish) [reply libs/utils/analytics.js]: Donezo

**Pull request merged**